src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger/config.jelly

```diff
@@ -194,8 +194,17 @@
         <f:entry title="${%Silent Mode}"
                  field="silentMode"
                  help="/plugin/gerrit-trigger/trigger/help-SilentMode.html">
-           <f:checkbox name="silentMode"
-                       checked="${it.silentMode}"/>
+            <table width="100%">
+                <f:optionalBlock title="" inline="true" name="silentMode"
```

Comment:
OK, I think I see what you are going for here, but then you need to handle `null` as a valid value in the code below as well and not default to something.